docs/batch-system-integration.md

```diff
@@ -0,0 +1,839 @@
+The following routes takes incoming jobs that have a `queue` attribute set to `"analy"` and routes them to the site's
+HTCondor batch system.
+Any other jobs will be sent to that site's PBS batch system.
+
+```hl_lines="2 6 7 12"
+JOB_ROUTER_ENTRIES @=jre
+[
+  TargetUniverse = 5;
+  name = "Route jobs to HTCondor";
+  Requirements = (TARGET.queue =?= "analy");
+]
+[
+  GridResource = "batch pbs";
+  TargetUniverse = 9;
+  name = "Route jobs to PBS";
+  Requirements = (TARGET.queue =!= "analy");
+]
+@jre
+```
+
+### Writing comments
+
+To write comments you can use `#` to comment a line:
+
+```hl_lines="5"
+JOB_ROUTER_ENTRIES @=jre
+[
+  TargetUniverse = 5;
+  name = "# comments";
+  # This is a comment
+]
+@jre
+```
+
+### Setting attributes for all routes
+
+To set an attribute that will be applied to all routes, you will need to ensure that `MERGE_JOB_ROUTER_DEFAULT_ADS` is set to `True` (check the value with [condor\_ce\_config\_val](troubleshoot-htcondor-ce#condor_ce_config_val)) and use the [set_](#setting-attributes) function in the `JOB_ROUTER_DEFAULTS`. The following configuration sets the `Periodic_Hold` attribute for all routes:
+
+```hl_lines="7"
+# Use the defaults generated by the condor_ce_router_defaults script.  To add
+# additional defaults, add additional lines of the form:
+#
+#   JOB_ROUTER_DEFAULTS = $(JOB_ROUTER_DEFAULTS) [set_foo = 1;]
+#
+MERGE_JOB_ROUTER_DEFAULT_ADS=True
+JOB_ROUTER_DEFAULTS = $(JOB_ROUTER_DEFAULTS) [set_Periodic_Hold = (NumJobStarts >= 1 && JobStatus == 1) || NumJobStarts > 1;]
+```
+
+### Filtering jobs based on…
+
+To filter jobs, use the `Requirements` attribute. Jobs will evaluate against the ClassAd expression set in the `Requirements` and if the expression evaluates to `TRUE`, the route will match. More information on the syntax of ClassAd's can be found in the [HTCondor manual](http://research.cs.wisc.edu/htcondor/manual/v8.6/4_1HTCondor_s_ClassAd.html). For an example on how incoming jobs interact with filtering in job routes, consult [this document](/compute-element/submit-htcondor-ce).
+
+When setting requirements, you need to prefix job attributes that you are filtering with `TARGET.` so that the job route knows to compare the attribute of the incoming job rather than the route’s own attribute. For example, if an incoming job has a `queue = "analy"` attribute, then the following job route will not match:
+
+```hl_lines="6"
+JOB_ROUTER_ENTRIES @=jre
+[
+  TargetUniverse = 5;
+  name = "Filtering by queue";
+  queue = "not-analy";
+  Requirements = (queue =?= "analy");
+]
+@jre
+```
+
+This is because when evaluating the route requirement, the job route will compare its own `queue` attribute to "analy" and see that it does not match. You can read more about comparing two ClassAds in the [HTCondor manual](http://research.cs.wisc.edu/htcondor/manual/v8.6/4_1HTCondor_s_ClassAd.html#SECTION00513300000000000000).
```

Comment:
```suggestion
This is because when evaluating the route requirement, the job route will compare its own `queue` attribute to "analy" and see that it does not match. You can read more about comparing two ClassAds in the [HTCondor manual](https://htcondor.readthedocs.io/en/stable/misc-concepts/classad-mechanism.html#old-classad-operators).
```